kernel/src/table_properties/deserialize.rs

```diff
@@ -0,0 +1,222 @@
+use crate::expressions::ColumnName;
```

Comment:
There's a `table_properties` directory. I think `table_properties.rs` should go in that directory as a `mod.rs`

Reply:
any reason instead of the current module setup?

Reply:
I hadn't realized that it was okay to have  it this way. I was only familiar with the `mod.rs` approach. Nothing wrong 👍